airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/WriteConfig.kt

```diff
@@ -3,14 +3,14 @@
  */
 package io.airbyte.cdk.integrations.destination.jdbc
 
-import io.airbyte.protocol.models.v0.DestinationSyncMode
+import io.airbyte.integrations.base.destination.typing_deduping.PostImportAction
 import java.time.Instant
 
 /**
  * Write configuration POJO (plain old java object) for all destinations extending
  * [AbstractJdbcDestination].
  */
-class WriteConfig
+data class WriteConfig
```

Comment:
I found WriteConfig as an unnecessary copy with ambiguous names and removed its reference in other destinations. 
I see you already made it almost identical to StreamConfig by removing ambiguous namings, any reason not to eliminate it completely in this part of PR and use StreamConfig instead ?

Reply:
2 reasons:
1) StreamConfig is in the typing-deduping submodule, and this is in db-destinations. so not every class that has access to WriteConfig has access to StreamConfig without moving stuff around
2) WriteConfig is created strictly after StreamConfig, and contains info that's dependent on the state of the destination DB (like the rawTableSuffix, and probably the tmpTableName), so switching one for the other is not as trivial as I would have liked